android/app/build.gradle

```diff
@@ -165,6 +166,15 @@ android {
             targetSdkVersion 29
             resValue "string", "build_config_package", "com.emurgo"
         }
+        nightly {
+            dimension "version"
+            applicationIdSuffix ".nightly"
+            applicationId 'com.emurgo'
+            minSdkVersion 21
+            compileSdkVersion = 29
+            targetSdkVersion 29
+            resValue "string", "build_config_package", "com.emurgo"
```

Comment:
not sure if `com.emurgo` needs to be changed here also

Reply:
nope https://github.com/luggit/react-native-config#advanced-android-setup